### PR TITLE
Fix Reporting page

### PR DESCRIPTION
**Pull request by ODORA0**
Make Reporting page more readable

### CODE REVIEW

File: reporting.md [hunk 1]
```diff
@@ -7,37 +7,43 @@ The reports are generated in the following ways:
 1. [BIRT Reports](reporting_birt.md) for facility level and some MoH HMIS reports 
 2. [Cohort Builder](reporting_cohort_builder.md) for custom data extraction in Excel for more detailed analysis
 
-USING THE COHORT BUILDER
+**USING THE COHORT BUILDER**
```
Comment (ssmusoke) [inline reporting.md]: @ODORA0 use two hashes for a heading 2

File: reporting.md [hunk 2]
```diff
@@ -7,37 +7,43 @@ The reports are generated in the following ways:
...
+![Click Encounter Tab](/assets/click_on_cohortbuilder_tab.png)
 
-1. 1. [UgandaEMR Reports](ugandaemr_reports.md) being developed to replace BIRT and provide a starting point for commonly used cohort builder report exports
+4.Click on search and view your results.
 
-USING THE DATA EXPORT FUNCTION
+*[UgandaEMR Reports](ugandaemr_reports.md) being developed to replace BIRT and provide a starting point for commonly used cohort builder report exports*
 
-1. Click on the "Administration" tab on the green navigation bar.
+**USING THE DATA EXPORT FUNCTION**
```
Comment (ssmusoke) [inline reporting.md]: @ODORA0 use 2 hashes ## for a heading 2

Comment (ODORA0) [reply reporting.md]: @ssmusoke Should I keep them bold?

Comment (ssmusoke) [reply reporting.md]: @ODORA0 that is not necessary the heading is sufficient

Comment (ODORA0) [reply reporting.md]: @ssmusoke Ok, left it as a heading anyway

File: reporting.md (hunk 1 above)
Comment (ODORA0) [reply reporting.md]: @ssmusoke Done

File: reporting.md [hunk 3]
```diff
@@ -7,37 +7,43 @@ The reports are generated in the following ways:
 1. [BIRT Reports](reporting_birt.md) for facility level and some MoH HMIS reports 
 2. [Cohort Builder](reporting_cohort_builder.md) for custom data extraction in Excel for more detailed analysis
 
-USING THE COHORT BUILDER
+## USING THE COHORT BUILDER
```
Comment (ssmusoke) [inline reporting.md]: @ODORA0 Headings should be title case not capital letters

File: reporting.md [hunk 4]
```diff
@@ -7,37 +7,43 @@ The reports are generated in the following ways:
...
+![Click Encounter Tab](/assets/click_on_cohortbuilder_tab.png)
 
-1. 1. [UgandaEMR Reports](ugandaemr_reports.md) being developed to replace BIRT and provide a starting point for commonly used cohort builder report exports
+4.Click on search and view your results.
 
-USING THE DATA EXPORT FUNCTION
+*[UgandaEMR Reports](ugandaemr_reports.md) being developed to replace BIRT and provide a starting point for commonly used cohort builder report exports*
 
-1. Click on the "Administration" tab on the green navigation bar.
+## USING THE DATA EXPORT FUNCTION
```
Comment (ssmusoke) [inline reporting.md]: @ODORA0 headings should be title case not all caps

File: reporting.md [hunk 5]
```diff
@@ -7,37 +7,43 @@ The reports are generated in the following ways:
...
+![Add New DataExport](/assets/add_new_dataexport.png)
 
-6. Save the Data Export and run it.
+4.Click on "Define Columns" and ensure that you have "simple column" selected
+
+![Click Define Columns](/assets/click_on_define_columns.png)
+
+5.Pick the "Encounter Date" value in the "first encounter info section of the drop down list as shown above and rename the Column Name to reflect the "Enrollment Date"
+
+6.Save the Data Export and run it.
```
Comment (ssmusoke) [inline reporting.md:48]: @ODORA0 There need to be setps on how to run the data export, if not add a section Running Data Export then put the text TBD

File: reporting.md (hunk 3 above)
Comment (ODORA0) [reply reporting.md]: @ssmusoke Done

File: reporting.md (hunk 4 above)
Comment (ODORA0) [reply reporting.md]: @ssmusoke Done

File: reporting.md (hunk 5 above)
Comment (ODORA0) [reply reporting.md:48]: @ssmusoke Added the Running Data Export section with a TBD

**Comment by ODORA0** [comment]
@ssmusoke I think this is ready for merging, unless if you have any reviews

**Pull request merged**